Use Xvfb for snapshots so we don't get an entire screen instead of
just a window (e.g. with multiwindow running).

--- origsrc/java-gnome-4.0.13/tests/screenshots/Harness.java	2009-08-26 22:19:52.000000000 -0500
+++ src/java-gnome-4.0.13/tests/screenshots/Harness.java	2009-11-29 05:22:16.696459900 -0600
@@ -66,7 +66,7 @@ import org.gnome.screenshot.Screenshot;
  */
 public final class Harness
 {
-    private static final boolean USE_VIRTUAL_DISPLAY = false;
+    private static final boolean USE_VIRTUAL_DISPLAY = true;
 
     public static void main(String[] args) throws IOException, InterruptedException {
         final String DISPLAY;
